build/makelib/helm.mk

```diff
@@ -62,7 +75,11 @@ helm.build: $(HELM_INDEX)
 define helm.dependency.update
 helm.dependency.update.$(1): $(HELM)
 	@echo === updating helm dependencies for $(1)
-	@cd $(HELM_CHARTS_DIR)/$(1) && $(HELM) dependency update
+	@if [ "$(call helm_needs_dependencies,$(1))" = "yes" ]; then \
+		cd $(HELM_CHARTS_DIR)/$(1) && $(HELM) dependency update; \
+	else \
+		echo "Dependencies already present, skipping"; \
+	fi
 helm.dependency.update: helm.dependency.update.$(1)
```

Comment:
was calling the `helm.dependency.update: helm.dependency.update.$(1)` resource consuming if its already present?